addons/mail_group/static/src/js/mail_group.js

```diff
@@ -27,56 +29,81 @@ publicWidget.registry.MailGroup = publicWidget.Widget.extend({
     /**
      * @private
      */
+    _toggleSubscribeButton: function(isSubscribe) {
+        this.el.querySelector('.o_mg_email_input_group').classList.toggle('d-none', isSubscribe);
+        this.el.querySelector('.o_mg_unsubscribe_btn').classList.toggle('d-none', !isSubscribe);
+    },
+
+    _updateMembersCount: function (action) {
+        if (!['added', 'removed'].includes(action)){
+            return;
+        }
+        var membersCount = parseInt(this.membersCountEl.textContent);
```

Comment:
Just to be safe with the parsing.
```suggestion
        const membersCount = parseInt(this.membersCountEl.textContent) || 0;
```